### PR TITLE
NETOBSERV-1503 Console plugin fields config

### PR DESCRIPTION
**Pull request by jpinsonneau**
## Description

Followup on https://github.com/netobserv/network-observability-operator/pull/550:
- [X] get fields from ConfigMap
- [X] force types when necessary

I decided to keep `loki.labels` instead of reusing fields `lokiLabel` since these are generated only for documentation purpose for now.

## Dependencies

n/a

## Checklist

If you are not familiar with our processes or don't know what to answer in the list below, let us know in a comment: the maintainers will take care of that.

* [X] Is this PR backed with a JIRA ticket? If so, make sure it is written as a title prefix _(in general, PRs affecting the NetObserv/Network Observability product should be backed with a JIRA ticket - especially if they bring user facing changes)._
* [ ] Does this PR require product documentation?
  * [ ] If so, make sure the JIRA epic is labelled with "documentation" and provides a description relevant for doc writers, such as use cases or scenarios. Any required step to activate or configure the feature should be documented there, such as new CRD knobs.
* [ ] Does this PR require a product release notes entry?
  * [ ] If so, fill in "Release Note Text" in the JIRA.
* [ ] Is there anything else the QE team should know before testing? E.g: configuration changes, environment setup, etc.
  * [ ] If so, make sure it is described in the JIRA ticket.
* QE requirements (check 1 from the list):
  * [X] Standard QE validation, with pre-merge tests unless stated otherwise.
  * [ ] Regression tests only (e.g. refactoring with no user-facing change).
  * [ ] No QE (e.g. trivial change with high reviewer's confidence, or per agreement with the QE team).

### CODE REVIEW

File: pkg/handler/config.go [hunk 1]
```diff
@@ -78,6 +78,15 @@ type QuickFilter struct {
 	Default bool `yaml:"default,omitempty" json:"default,omitempty"`
 }
 
+type FieldConfig struct {
+	Name        string `yaml:"name" json:"name"`
+	Type        string `yaml:"type" json:"type"`
+	Description string `yaml:"description" json:"description"`
+	// This flag is for documentation only. Use loki.labels instead
```
Comment (jotak) [inline pkg/handler/config.go:85]: perhaps it can be omitted then, in this declaration? To avoid any confusion

**Review by jotak — APPROVED** [review]
code lgtm!

File: pkg/handler/config.go (hunk 1 above)
Comment (jpinsonneau) [reply pkg/handler/config.go]: sure :+1: 
https://github.com/netobserv/network-observability-console-plugin/pull/476/commits/2fbb838e1fa395a506aa08ceb12b39f0e8433f87

**Pull request merged**